dashboards/component/chart/serializers.py

```diff
@@ -11,21 +11,18 @@
 
 from dashboards.meta import ClassWithMeta
 
+from dashboards.component.filters import MyFilter
 
 class ModelDataMixin:
-    """
-    gets data from a django model and converts to a pandas dataframe
-    """
-
     class Meta:
         fields: Optional[List[str]] = None
         model: Optional[Model] = None
+        filter_component = MyFilter
```

Comment:
this shouldn't be defined on `Meta` it needs to be passed into the Serializers `get_data()` / `get_queryset()` method from the dashboard